packages/extension/src/components/pages/Login/Login.tsx

```diff
@@ -61,12 +61,20 @@ import { useWallet } from '../../../contexts';
 import { useKeyUp } from '../../../hooks/useKeyUp';
 import { loadData } from '../../../utils';
 import { loadRememberSessionState, saveRememberSessionState } from '../../../utils/login';
+import {
+  loadFromChromeLocalStorage,
+  saveInChromeLocalStorage
+} from '../../../utils/storageChromeLocal';
 import { Logo } from '../../atoms/Logo';
 
+const maxAttempts = 5;
```

Comment:
```suggestion
const maxAttempts = 5;
const DELAY_RETRY_MINS = 15;
```

Reply:
Thanks for the tip, it's better to have number in variables instead of magic numbers